packages/fscomponents/src/components/ReviewItem.tsx

```diff
@@ -98,38 +99,16 @@ export class ReviewItem extends Component<ReviewItemProps, ReviewItemState> {
 
   renderSyndicatedIndicator = (): JSX.Element | undefined => {
     if (this.props.syndicationSource && this.props.syndicationSource.LogoImageUrl) {
-
-      Image.getSize(
-        this.props.syndicationSource.LogoImageUrl,
-        this.getImageSizeSuccess,
```

Comment:
Since you are removing the reference here, you should be able to remove the function, too.

Reply:
Good Catch. Removed.